src/Tribe/Utils/DST.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+
+class Tribe__Events__Utils__DST {
```

Comment:
Just felt "general-purpose enough" to me. Even if we initially use it in just one place, it's been written cleanly and in a way that doesn't bind it to any one of our plugins. So, totally up to you, just wanted to moot the idea :+1: